search-sentiments-map/index.yaml

```diff
@@ -1,6 +1,6 @@
 indexes:
 
-- kind: CustomSearchTopic
+- kind: WorldDataByTopic
```

Comment:
Why did you change this? I thought Carmen was using it for the custom search results data

Reply:
Because we are changing the entity name